build_scripts/colab-tests/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
```

Comment:
I've made this its own poetry package as it requires `poetry` as a dependency which:

a) is clashing with some of our other deps
b) is probably going to cause headaches for people using conda